sites.json

```diff
@@ -2402,6 +2402,15 @@
             "gumtree.com"
         ]
     },
+    
+    {
+        "name": "Gyazo",
+        "url": "https://gyazo.com/settings",
```

Comment:
Is it clear on this page how to delete the account? If not, some notes should be added. (If it is, this is fine; I just don't personally know myself)